sources/osg/RenderStage.js

```diff
@@ -265,10 +265,21 @@ MACROUTILS.createPrototypeObject(
             state.applyAttribute(this.viewport);
 
             // fragment clipping
+
             if (this.camera) {
```

Comment:
All this the code in this.camera is useless, it was necessary if we cleared by sub regions but if we dont do that we dont need to apply it here